### DISCUSSION TITLE
intallation process

### DISCUSSION THREAD

**Discussion by rajeshkandula369** [Security-Onion-Solutions/securityonion#12291] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Import

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

12

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

i am importing the security onion iso file in vmware workstation ,I got stucked at this point which is shown in the following screenshot.i am at starting stage of installation .
![11](https://github.com/Security-Onion-Solutions/securityonion/assets/158308916/02b80db1-cc96-45df-b980-561dfc435cec)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Those are kernel messages, you should be able to login with your username and password. https://docs.securityonion.net/en/2.4/console.html

**Comment by rajeshkandula369** [comment]
Thanks, sir
